tests/postgres_tests/models.py

```diff
@@ -144,6 +144,15 @@ class RangesModel(PostgreSQLModel):
     dates_inner = DateRangeField(blank=True, null=True)
 
 
+class RangesModelGeneratedField(PostgreSQLModel):
+    ints = IntegerRangeField(blank=True, null=True)
+    ints_generated = models.GeneratedField(
+        expression=models.F("ints"),
+        output_field=IntegerRangeField(null=True),
+        db_persist=True,
+    )
```

Comment:
I think this is needed
```suggestion
    )

    class Meta:
        required_db_features = {"supports_stored_generated_columns"}
```

Reply:
Done, thanks!

Reply:
It appears that adding this causes the tests to fail in databases that don't support stored generated fields. None of the other models in postgres_tests/models.py have similar Meta flags, so I will leave this out.